plugins/woocommerce/client/admin/client/settings-payments/components/payment-method-list-item/payment-method-list-item.tsx

```diff
@@ -0,0 +1,124 @@
+/**
+ * External dependencies
+ */
+import { decodeEntities } from '@wordpress/html-entities';
+import { type RecommendedPaymentMethod } from '@woocommerce/data';
+import { ToggleControl } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import sanitizeHTML from '~/lib/sanitize-html';
+
+type PaymentMethodListItemProps = {
+	method: RecommendedPaymentMethod;
+	paymentMethodsState: Record< string, boolean >;
+	setPaymentMethodsState: ( state: Record< string, boolean > ) => void;
+	isExpanded: boolean;
+};
+
+export const PaymentMethodListItem = ( {
+	method,
+	paymentMethodsState,
+	setPaymentMethodsState,
+	isExpanded,
+	...props
+}: PaymentMethodListItemProps ) => {
+	if ( ! method.enabled && ! isExpanded ) {
+		return null;
+	}
+
+	return (
+		<div
+			id={ method.id }
+			className="woocommerce-list__item woocommerce-list__item-enter-done"
+			{ ...props }
+		>
+			<div className="woocommerce-list__item-inner">
+				{ method.id !== 'payment_request' && (
```

Comment:
what does `payment-request` ID mean

Reply:
I’ve updated the payment_request capability to be renamed as card_payments.